cogs/whisper.py

```diff
@@ -14,8 +16,14 @@ def __init__(self, bot):
     async def psst(self, ctx, *,message: str):
         """Psst - PM the bot with the !whisper prefix,
         along with your message."""
+        
+        random_animals = ['Ferret', 'Donkey', 'Robin', 'Platypus', 'Moose', 'Panda', 'Aardvark', 'Dog', 'Ant', 'Cheetah', 'Chicken', 'Duck', 'Eagle', 'Gecko', 'Lion', 'Pig', 'Wolf'] 
+        random_adjectives = ['Alert', 'Alive', 'Amused', 'Clever', 'Crazy', 'Dizzy', 'Jolly', 'Brave', 'Tired', 'Busy', 'Bored']
+
+        seed = os.environ.get('WHISPERHASH')
```

Comment:
no need to constantly allocate lists `random_animals`, `random_adjectives` (and deallocate) with every whisper, these allocated in class `__init__` instead, the same aplies for seed.

Reply:
We can make these class variables instead, right?

Reply:
I have added the variables to the class initialization